### DISCUSSION TITLE
Sensitivity analysis

### DISCUSSION THREAD

**Discussion by udiwekar** [IDAES/idaes-pse#507] [General]
Can you do sensitivity analysis in IDAES

**Comment by blnicho** [comment]
Yes, there are a number of tools under development in both IDAES-PSE and Pyomo for handling different aspects of sensitivity analysis. You can find more information on these tools at the links below:

Uncertainty Propagation Toolbox: https://idaes-pse.readthedocs.io/en/latest/user_guide/modeling_extensions/uncertainty_propagation/index.html
Sensitivity Toolbox: https://pyomo.readthedocs.io/en/latest/contributed_packages/sensitivity_toolbox.html 

If you provide us with more information on your use-case we can give you more specific guidance on which tools might be the most relevant.
